src/scripts/main.js

```diff
@@ -1,3 +1,65 @@
 'use strict';
 
-// write code here
+const queryList = document.querySelector('ul').children;
+
+const getEmployees = function(list) {
+  const arrayList = [...list].map(item => {
+    const itemEmployee = {
+      name: item.innerHTML.trim(),
+      position: item.getAttribute('data-position'),
+      salary: item.getAttribute('data-salary'),
+      age: item.getAttribute('data-age'),
+    };
+
+    return itemEmployee;
+  });
+
+  return arrayList;
+};
+
+const listForSort = getEmployees(queryList);
+
+const sortList = function(list) {
+  list.sort(function(a, b) {
+    let aSort = a.salary.slice(1, 10).split(',');
+
+    aSort = Number(aSort.join(''));
+
+    let bSort = b.salary.slice(1, 10).split(',');
+
+    bSort = Number(bSort.join(''));
+
+    if (aSort > bSort) {
+      return -1;
+    }
+
+    if (aSort < bSort) {
+      return 1;
+    }
+
+    return 0;
+  });
+
+  return list;
+};
+
+sortList(listForSort);
```

Comment:
```suggestion
const sortList = function(list) {
  list.sort(function(a, b) {
    let aSort = a.salary.slice(1, 10).split(',');

    aSort = Number(aSort.join(''));

    let bSort = b.salary.slice(1, 10).split(',');

    bSort = Number(bSort.join(''));

    if (aSort > bSort) {
      return -1;
    }

    if (aSort < bSort) {
      return 1;
    }

    return 0;
  });

  return list;
};

sortList(getEmployees(queryList));
```

Reply:
It doesn't work either, the list is used below in a loop. Or there, too, must be changed to getEmployees?